scripts/adaptive-sampling-integration-test.sh

```diff
@@ -2,16 +2,69 @@
 
 set -euf -o pipefail
 
-# This script is currently a placeholder.
+# Function to start Jaeger with adaptive sampling
+start_jaeger() {
+  echo "Starting Jaeger with adaptive sampling..."
+  SAMPLING_STORAGE_TYPE=memory SAMPLING_CONFIG_TYPE=adaptive go run -tags=ui ../cmd/all-in-one --log-level=debug
```

Comment:
you are not passing the desired initial sampling rate, as you mentioned in the plan